spa/src/styles/themes.js

```diff
@@ -59,7 +59,14 @@ export const donationPageBase = merge({}, revEngineTheme, {
     lg: '1100px',
     xl: '1300px'
   },
-  font: { body: "'Montserrat', sans-serif", heading: "'Montserrat', sans-serif" }
+  font: { body: "'Montserrat', sans-serif", heading: "'Montserrat', sans-serif" },
+  colors: {
+    cstm_mainHeader: revEngineTheme.colors.white,
+    cstm_mainBackground: '#faf8f8',
+    cstm_formPanelBackground: revEngineTheme.colors.white,
+    cstm_CTAs: revEngineTheme.colors.primary,
+    cstm_ornaments: '#2b2869'
+  }
```

Comment:
Now that I see this in action, I think we might as well add these too, even though they're not in the example provided:

```suggestion
    cstm_ornaments: '#2b2869'
    cstm_inputBackground: revEngineTheme.colors.white,
    cstm_inputBorder: revEngineTheme.colors.black
  }
```